CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Screen API Coordinator/GiniScreenAPICoordinator+Analysis.swift

```diff
@@ -33,27 +34,29 @@ extension GiniScreenAPICoordinator {
         }()
         
         if isCameraViewControllerLoaded {
-            imageAnalysisNoResultsViewController = ImageAnalysisNoResultsViewController()
-            imageAnalysisNoResultsViewController.setupNavigationItem(usingResources: backButtonResource,
-                                                                     selector: #selector(backToCamera),
-                                                                     position: .left,
-                                                                     target: self)
+            viewModel = NoResultScreenViewModel { [weak self] in
+                self?.backToCamera()
+            } manuallyPressed: { [weak self] in
+                //TODO: the same as cancel
+                self?.closeScreenApi()
+            } cancelPressed: { [weak self] in
+                self?.backToCamera()
+            }
+            
         } else {
-            imageAnalysisNoResultsViewController = ImageAnalysisNoResultsViewController(bottomButtonText: nil,
-                                                                                        bottomButtonIcon: nil)
-            imageAnalysisNoResultsViewController.setupNavigationItem(usingResources: closeButtonResource,
-                                                                     selector: #selector(closeScreenApi),
-                                                                     position: .left,
-                                                                     target: self)
+            viewModel = NoResultScreenViewModel { [weak self] in
+                //TODO: check if this make sense
+                self?.closeScreenApi()
+            } manuallyPressed: { [weak self] in
+                //TODO: check if this make sense
+                self?.closeScreenApi()
+            } cancelPressed: { [weak self] in
+                self?.closeScreenApi()
+            }
         }
+        viewController = NoResultScreenViewController(giniConfiguration: giniConfiguration, errorType: .image, viewModel: viewModel)
```

Comment:
We don't have an option here like:
 NoResultScreenViewController(giniConfiguration: giniConfiguration, errorType: .pdf, viewModel: viewModel)
 
  It doesn't matter if it's an image or pdf I see always the same screen.

 
![IMG_ED04C4D81FB4-1](https://user-images.githubusercontent.com/3911816/186948428-3138de30-70e6-4ed0-b6ce-0a6d1c63e0e9.jpeg)

For PDF we need to show:

 https://www.figma.com/file/m4OtCvxWxPMeB0Z5k4Aeow/Gini-Photo-Payment-%2F-iOS?node-id=606%3A9030
 with the title "No Results" and error text "Could not retrieve any information
from your documents."

Reply:
To show/ hide buttons criteria will be solved on Monday 🙈

Reply:
@zladzeyka Ok, I have fixed the issue with showing specific type. The correct navbar title is resolved in https://github.com/gini/gini-mobile-ios/pull/134 branch